automation_tools/__init__.py

```diff
@@ -373,41 +373,28 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
             print('Was not possible to fetch interface information')
             sys.exit(1)
 
-    if os.environ.get('SATELLITE_VERSION') in ('6.0', '6.1'):
-        proxy = 'capsule'
-    else:
-        proxy = 'foreman-proxy'
-
     installer_options = {
-        '{0}-dns'.format(proxy): 'true',
-        '{0}-dns-forwarders'.format(proxy): forwarders,
-        '{0}-dns-interface'.format(proxy): interface,
-        '{0}-dns-zone'.format(proxy): domain,
-        '{0}-dhcp'.format(proxy): 'true',
-        '{0}-dhcp-interface'.format(proxy): interface,
-        '{0}-tftp'.format(proxy): 'true',
-        '{0}-tftp-servername'.format(proxy): hostname,
-        '{0}-register-in-foreman'.format(proxy): 'true',
+        'foreman-proxy-dns': 'true',
+        'foreman-proxy-dns-forwarders': forwarders,
+        'foreman-proxy-dns-interface': interface,
+        'foreman-proxy-dns-zone': domain,
+        'foreman-proxy-dhcp': 'true',
+        'foreman-proxy-dhcp-interface': interface,
+        'foreman-proxy-tftp': 'true',
+        'foreman-proxy-tftp-servername': hostname,
+        'foreman-proxy-register-in-foreman': 'true',
     }
 
-    # Handles enabling puppet support
-    if not os.environ.get('SATELLITE_VERSION') in ('6.3', '6.4'):
```

Comment:
Here we missed to add newer versions (6.5, 6.6)
So when considering removal of this code read this as 
```
if not os.environ.get('SATELLITE_VERSION') in ('6.3', '6.4', '6.5', '6.6'):
```

Reply:
Anyway this options aren't needed for 6.3 and older since they are both true by default (tested)